ui/src/common/QualityInfo.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import Chip from '@material-ui/core/Chip'
+
+export const QualityInfo = (props) => {
+  let { suffix, bitRate } = props.record
```

Comment:
What about:
```js
export const QualityInfo = ({record, ...rest}) => {
```

and then passing `rest` to the `Chip` component, allowing for external customization:
```js
<Chip size="small" variant="outlined" label={info} {...rest}/>
```

Reply:
But its not throwing an exception when i played songs from playlist.

Reply:
@deluan its the problem in ``` PlayerToolbar.js``` . I am thinking why its not working. here is the screenshot of ```data``` which i am passing it as``` record``` prop in ```       <QualityInfo record={data} />```. It has both suffix and bitRate.
![image](https://user-images.githubusercontent.com/53482681/112872388-f48d3880-90dd-11eb-9d79-5d386ddbdfd8.png)